test/built-ins/Array/prototype/copyWithin/coerced-values.js

```diff
@@ -0,0 +1,92 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 22.1.3.3
+description: >
+  Arguments are coerced to integer values.
+info: >
+  22.1.3.3 Array.prototype.copyWithin (target, start [ , end ] )
+
+  ...
+  5. Let relativeTarget be ToInteger(target).
+  ...
+  8. Let relativeStart be ToInteger(start).
+  ...
+  11. If end is undefined, let relativeEnd be len; else let relativeEnd be
+  ToInteger(end).
+  ...
+includes: [compareArray.js]
+---*/
+
+assert(
+  compareArray(
+    [0, 1, 2, 3].copyWithin(undefined, undefined),
```

Comment:
To be more consistent with the rest of your tests, these assertions should be grouped into three separate files: one for step 5, one for step 8, and one for step 11.